### ISSUE TITLE
Not  restricticted iothreads and other qemu threads

### ISSUE THREAD

**Issue by jcvenegas** [kata-containers/runtime#1430]
**Impact**: it helps to keep the promise that kata containers only uses cpus that were assigned to it.
**Description**:
Not all Kata threads are cpuset restrictricted, only vcpus threads are added, but if threads are not added as it degrade the performance for IO and network.
**Possible solutions** :
Document that those qemu threads will be free to be scheduled in all CPUs as they are considered system threads for us.
Add the rest of threads to a “sandbox cgroup” (a cgroup that is the union of all containers of a pod). As this could have a performance degradation  add an option to kata containers configuration to enable or disable this.

This is related with cpumanager static policy.

Tl;dr; Kubernetes scheduled workloads that do not belong to that cpuset should not land on those CPUs. This includes kata’s qemu threads. Also if a Kata pod is using a cpuset, then it should stay within the cpuset. The goal of CPU set is guaranteed performance (not necessarily high performance)
